en/docs/design-the-project/explore-the-architecture-view.md

```diff
@@ -125,9 +119,30 @@ If these data types are Ballerina records, click on them to access the compositi
 
 <img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/level-two-to-type-composition.gif" class="cInlineImage-full"/>
 
-### Type Diagram
+### Type diagram
+
+The type diagram provides a comprehensive view of all the Ballerina record types defined in the project workspace and their associations. This can be accessed via the **Architecture View** command, as shown below.
+
+<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/type-diagram/type-diagram.gif" class="cInlineImage-full"/>
```

Comment:
Why not having the same gcp demo in the gif? And no need to have two images/gifs to this section. Always stick to one example.

Reply:
Updated.